medusa/clients/torrent/deluge_client.py

```diff
@@ -37,6 +37,8 @@ def read_torrent_status(torrent_data):
         for i in details['files']:
             # Check if media was processed
             # OR check hash in case of RARed torrents
+            if not is_media_file(i['path']):
+                continue
             if is_already_processed_media(i['path']) or is_info_hash_processed(info_hash):
```

Comment:
Had to change this because when it wasn't a media file, the `is_info_hash_processed` is called. But it shouldn't